lib.php

```diff
@@ -291,3 +320,15 @@ function scormremote_pluginfile($course, $cm, $context, $filearea, $args, $force
 
     send_stored_file($file, $lifetime, 0, false, $options);
 }
+
+/**
+ * Manipulate the string to token replace the debug log level.
+ * @param string $layer3content
+ * @return void
+ */
+function applyloglevel(string &$filecontent)
+{
+    $re = '/(logLevel:\s*)(\s\d,)/';
```

Comment:
I would make this very explicit so you don't accidentally touch some unrelated js. I've make a token in the js file like LOGLEVEL_TOKEN and put a comment next to it in the js file so its clear it gets swapped out

Reply:
Done